doc/DeveloperGuide.md

```diff
@@ -94,6 +94,7 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Users are expected to adhere to the input format whilst entering commands
```

Comment:
Not sure if this qualifies as NFR. this is more like instruction to the user.
Please go through the lecture material on NFR for better understanding and try to come up with more NFRs.